Source/WebCore/platform/gamepad/cocoa/GameControllerGamepadProvider.mm

```diff
@@ -39,6 +39,19 @@
 
 #import "GameControllerSoftLink.h"
 
+#if USE(APPLE_INTERNAL_SDK) && __has_include(<WebKitAdditions/GameControllerAdditions.mm>)
```

Comment:
I think this really should be *just* `USE(APPLE_INTERNAL_SDK)`

I know you're checking the "has_include" to help internal SDK people build who might not have updated sources/etc, but it introduces an invalid and potentially sneaky state of "What if someone refactors WebKitAdditions and/or gets it out of sync on a branch or whatever", which could inadvertently give the wrong behavior to the internal WebKit build.

We *should* actually break that build if <WebKitAdditions/GameControllerAdditions.mm> becomes unavailable, or changes names, etc.

Reply:
Makes sense! I have updated the PR.